test/mountain_view/component_test.rb

```diff
@@ -17,15 +17,95 @@ def test_styleguide_stubs
     component = MountainView::Component.new("header")
 
     expected_stub = {
-      id: 1,
-      title: "20 Mountains you didn't know they even existed",
-      subtitle: "Buzzfeed title"
-    }
+                      :meta=>"There is this different classes",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Surrounding space missing for operator `=>`.